sormas-backend/src/main/java/de/symeda/sormas/backend/caze/CaseFacadeEjb.java

```diff
@@ -1347,9 +1369,14 @@ public Long countCasesForMap(RegionReferenceDto regionRef, DistrictReferenceDto
 		}
 
 		Expression<Number> maxReportDate = cb.max(caze.get(Case.REPORT_DATE));
+		Expression<Number> maxCreationDate = cb.max(caze.get(Case.CREATION_DATE));
 		cq.multiselect(caze.get(Case.DISEASE), districtJoin, maxReportDate);
```

Comment:
Don't you also need to select the creation date in order to be able to sort by it?

Reply:
Don't need to select it if I am using just for sorting. In fact, we could remove `maxReportDate` from `multiselect `as well.

Reply:
Ah, interesting, didn't know about that! Yes, I think you could just remove maxReportDate as well then :)